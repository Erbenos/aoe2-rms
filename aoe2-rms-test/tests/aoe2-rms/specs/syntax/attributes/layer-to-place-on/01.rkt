#lang aoe2-rms
; Place rocks on a small patch of layered snow within a larger desert area.

<LAND-GENERATION>
(create-land
 (terrain-type 'DESERT)
 (number-of-tiles 500)
 (land-position 50 50))

<TERRAIN-GENERATION>
(create-terrain 'SNOW
                (base-terrain 'DESERT)
                (number-of-tiles 20)
                (terrain-mask 1))

<OBJECTS-GENERATION>
(create-object 'ROCK
               (number-of-objects 300)
               (layer-to-place-on 'DESERT))
