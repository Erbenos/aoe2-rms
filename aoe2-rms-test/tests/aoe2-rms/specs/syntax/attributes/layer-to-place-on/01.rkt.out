<LAND_GENERATION>
create_land {
  terrain_type DESERT
  number_of_tiles 500
  land_position 50 50
}
<TERRAIN_GENERATION>
create_terrain SNOW {
  base_terrain DESERT
  number_of_tiles 20
  terrain_mask 1
}
<OBJECTS_GENERATION>
create_object ROCK {
  number_of_objects 300
  layer_to_place_on DESERT
}
